cypress/e2e/login.cy.js

```diff
@@ -2,10 +2,35 @@
 
 describe('Login page', () => {
   beforeEach(() => {
-    
+    cy.visit('https://the-internet.herokuapp.com/login')
   });
 
-  it('', () => {
-    
+  it('Login with valid creds', () => {
```

Comment:
the name of each of your test cases should continue the describe name
```suggestion
  it('should allow to login with valid credentials', () => {
```